decidim-admin/config/locales/en.yml

```
@@ -965,6 +966,11 @@ en:
           help: These tokens are used to publicly share this unpublished resource to any user. They will be hidden when the resource is published. Click on the token's share icon to visit the shareable URL.
           title: Share tokens
       shared:
+        adjacent_navigation:
+          next: Next
+          next_title: Link to the next item
+          previous: Previous
+          previous_title: Link to the previous item
```

I think we can drop the "link" as this is shown in an anchor element. I've just checked it out how we used to do it in the frontend and we just said "Next item"/"Previous item" (see #13011)

```suggestion
          next_title: Next item
          previous: Previous
          previous_title: Previous item
```